api/HTMLVideoElement.json

```diff
@@ -555,7 +555,7 @@
               "version_added": "12"
             },
             "firefox": {
-              "version_added": "4"
+              "version_added": "3.6"
```

Comment:
This is the "poster" attribute, and the entry in video.json already says 3.6. Good!